db/src/main/scala/tamer/db/config.scala

```diff
@@ -2,26 +2,23 @@ package tamer
 package db
 
 import cats.syntax.all._
-import ciris.refined.refTypeConfigDecoder
 import ciris.{ConfigException, env}
-import eu.timepit.refined.types.numeric.PosInt
-import eu.timepit.refined.types.string.NonEmptyString
-import zio.interop.catz.{taskConcurrentInstance, zioContextShift}
+import zio.interop.catz._
 import zio.{Has, Layer, Task}
 
-final case class ConnectionConfig(driver: NonEmptyString, uri: UriString, username: NonEmptyString, password: Password)
-final case class QueryConfig(fetchChunkSize: PosInt)
+final case class ConnectionConfig(driver: String, uri: String, username: String, password: String)
+final case class QueryConfig(fetchChunkSize: Int)
```

Comment:
removing refined types

Reply:
Yes, I think we learned that offering refined types to end users should be done more conservatively. Certainly it took a big toll and didn't give a lot back in terms of security or dev experience.